ouf.lua

```diff
@@ -346,6 +346,17 @@ local function initObject(unit, style, styleFunc, header, ...)
 			func(object)
 		end
 
+		--[[ frame.IsPingable
+		This boolean can be set to false to disable the frame from being pingable. Enabled by default.
+		--]]
+		--[[ Override: frame:GetContextualPingType()
+		Used to define which contextual ping is used for the frame.
+
+		By default this wraps `C_Ping.GetContextualPingTypeForUnit(UnitGUID(frame.unit))`.
+		--]]
+		object:SetAttribute('ping-receiver', true)
```

Comment:
Just wanted to let you know that this line is breaking frames when they're spawned in combat.

Reply:
Just to make it easier for me to test this, how did you encounter this? E.g. when did you get frames spawned during combat?

Reply:
I was on a world boss and it broke when players joined the raid.